projects/elife.yaml

```diff
@@ -2047,17 +2047,18 @@ data-pipeline:
                 tables:
                     # bit of repetition here
                     380_datascience_early_career_researchers:
-                        schema: https://raw.githubusercontent.com/elifesciences/data-pipeline-ejp-csv-deposit/master/schemas/380_datascience_early_career_researchers.json
+                        schema: bigquery/schemas/380_datascience_early_career_researchers.json
```

Comment:
we decided for a separate repository in https://github.com/elifesciences/data-pipeline/pull/2 what has changed that makes us put the files in here? Reading the linking ticket now

Reply:
I should have put a link to the ticket in the PR, sorry, I explain my reasoning there.